### PR TITLE
docs: add v2023.2.4 release notes

### PR DESCRIPTION
**Pull request by herbetom**
Based on the work done in https://md.darmstadt.ccc.de/gluon-v2023.2.4-rn

A preview can be found here: https://gluon--3345.org.readthedocs.build/en/3345/releases/v2023.2.4.html

Contains the following changes: https://github.com/freifunk-gluon/gluon/compare/v2023.2.3...8d04b5e4961e7c0a779bc8755142b65cca19f72e

### CODE REVIEW

File: docs/releases/v2023.2.4.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- Sophos
+
+  - AP15
```
Comment (kpanic23) [inline docs/releases/v2023.2.4.rst:12]: Is the Sophos AP15 supposed to be on the list? I thought "broken" devices aren't listed?
https://github.com/freifunk-gluon/gluon/pull/3126#discussion_r1438419927

**Comment by neocturne** [comment]
Should we try to get https://github.com/freifunk-gluon/ecdsautils/pull/1 merged before the release, or do we not really care because the only target that is known to be affected is BROKEN anyways?

**Comment by blocktrron** [comment]
@neocturne I've reviewed the PR in question. I don't think it matters to much, given the target is effectively not in use.

**Comment by neocturne** [comment]
> @neocturne I've reviewed the PR in question. I don't think it matters to much, given the target is effectively not in use.

Makes sense

File: docs/releases/v2023.2.4.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Added hardware support
+----------------------
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- NETGEAR
+
+  - EX6130
```
Comment (neocturne) [inline docs/releases/v2023.2.4.rst:12]: EX6130 is mt7620, not mt7621.

Comment (herbetom) [reply docs/releases/v2023.2.4.rst:12]: good catch. Fixed it

**Pull request merged**